bot.go

```diff
@@ -242,11 +245,11 @@ func (b *Bot) Run() error {
 
 	// Now that we have a client, set up debug callbacks
 	b.client.Reader.DebugCallback = func(line string) {
-		b.log.Debug("<-- ", line)
+		b.log.Debug("<-- ", strings.TrimSpace(line))
```

Comment:
Could we just make these `strings.TrimRight(line, "\r\n")`? I think that should still work and it won't drop spaces from the end of lines if they should be there.